src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -349,6 +361,12 @@ private static void QueryUniqueChannelBinding(SafeSslHandle context, SafeChannel
                 throw CreateSslException(SR.net_ssl_get_channel_binding_token_failed);
             }
 
+            if (!sessionReused)
+            {
+                // The operation may have left errors on the queue
+                Crypto.ErrClearError();
```

Comment:
According to the man page for ssl_session_reused, it returns:
```
0: A new session was negotiated.
1: A session was reused.
```
Why do we only need to clear errors in the negotiated case rather than the reused case?